integration_tests/integration/viewAndPrintLicence.cy.ts

```diff
@@ -320,4 +320,19 @@ context('View and print licence', () => {
     const viewLicencePage = viewCasesList.clickALicence()
     viewLicencePage.checkIfElectronicMonitoringAdditionalInformationDoesNotExist()
   })
+
+  it('Should navigate back to the "Future Release" tab when clicking the "Back" link', () => {
```

Comment:
Should this be called `Should navigate back to the current active tab when clicking the "Back" link` as that's the behaviour we want? I read this as it will always go back to the "Future Release" tab but I think we only want that if it's the active tab?